harvester/cli.py

```diff
@@ -81,33 +81,58 @@ def main(ctx, host, output_file, verbose):
     help="Optional: SetSpec of set to be harvested. Limits harvest to records in the "
     "provided set.",
 )
+@click.option(
+    "-sr",
+    "--skip-record",
+    envvar="RECORD_SKIP_LIST",
+    multiple=True,
+    show_default=True,
+    help="Optional: OAI-PMH identifier of record to skip during harvest. Can be "
```

Comment:
May be worth reiterating the expected delimiter and that option this can only be used with `get`, or guiding users to `README.md` for more detail